osu.Game/Graphics/UserInterface/OsuMenu.cs

```diff
@@ -68,7 +68,9 @@ protected override void AnimateClose()
             if (!TopLevelMenu && wasOpened)
                 menuSamples?.PlayCloseSample();
 
-            this.FadeOut(300, Easing.OutQuint);
+            this.Delay(50)
+                .FadeOut(300, Easing.OutQuint);
```

Comment:
Did / do you want this change to also apply to context menus? Asking because `OsuContextMenu` overrides `AnimateClose()` and does not call base:

https://github.com/ppy/osu/blob/b60a8341e4eeaddfb6e89e53edc246544c17156f/osu.Game/Graphics/UserInterface/OsuContextMenu.cs#L60-L68

so closes on context menus are still not delayed on this branch as it is.

Reply:
Have applied this, good catch.